README.md

```diff
@@ -13,7 +13,8 @@ A highly functional theme that adapts to the reader's preferences. Let them read
 
 # Search
 
-1. Generate a content API key in `Integrations` section which will be used to fetch posts from your site.
+1. Navigate to the `Integrations` and click on `Add custom integration`. 
+2. Copy the content API key; this will be used to fetch posts from your site.
 2. Insert the generated key in `Code injection > Site Header` field.
```

Comment:
Could you number this to `3`, because there are now two instances of `2`?

Reply:
Fixed in #23 -- odd this one seems to have detached from the head and I can't figure out why. 

Cheers